test/Microsoft.ML.Core.Tests/UnitTests/TestEntryPoints.cs

```diff
@@ -5495,7 +5511,11 @@ public void TestCrossValidationMacroMulticlassWithWarnings()
             }
         }
 
+#if INNER_LOOP
```

Comment:
Can you please investigate whether is possible to skip the test at runtime instead of disabling it at compile time?